tests/devices/test_default_qubit.py

```diff
@@ -181,32 +181,45 @@ def test_apply_operation_single_wire_no_parameters_inverse(
         assert np.allclose(qubit_device_1_wire._state, np.array(expected_output), atol=tol, rtol=0)
 
     test_data_two_wires_no_parameters = [
-        (qml.CNOT, [1, 0, 0, 0], [1, 0, 0, 0]),
-        (qml.CNOT, [0, 0, 1, 0], [0, 0, 0, 1]),
+        (qml.CNOT, [1, 0, 0, 0], [1, 0, 0, 0], [1, 0, 0, 0]),
```

Comment:
This is a nice solution, though it makes the data more verbose. Since for now, there's only one gate where the expected output of the inverse computation differs from the non-inverse case, it would be better to treat cases using that operation separately and reverting back to the original state of this list.

Two separate lists e.g., `test_iswap_data` and `test_iswap_data_inverse` could be defined to add the cases for `ISWAP`. Then, `test_data_two_wires_no_parameters + test_iswap_data` and `test_data_two_wires_no_parameters + test_iswap_data_inverse` could be passed to the corresponding tests (merging the two lists could also come right before the test, e.g., with `all_two_qubit_no_param = test_data_two_wires_no_parameters + test_iswap_data` and then passing `all_two_qubit_no_param` to the first test).

Reply:
I agree, was thinking whether to keep it separate and concatenate too...